ComputerGuesses.js

```diff
@@ -0,0 +1,140 @@
+  
+    // if statement that is for when the user says no (or any word starting with 'n' for that matter, try to figure out fixing this.)
+    if (capitalize(response).charAt(0) === "N") {
+
+      //allow the computer to ask if it is higher or lower
+      let highOrLow = await ask("Is your number higher or lower? ")
+
+      // while loop so that if the response to higher or lower isnt a proper response, it will tell the player and ask again 
+      while (capitalize(highOrLow).charAt(0) !== "H" && capitalize(highOrLow).charAt(0) !== "L") {
+        highOrLow = await ask("That is not Higher or Lower please try again.\nWas your number higher or lower? ")
+      }
+
+      // if statement that if the response is higher, it will remember and guess appropriately.
+      if (capitalize(highOrLow).charAt(0) === "H") { 
+
+        // redefine the low number to restrict the low side of the range.
+        lowNum = randNum + 1
+       
+        // if the high number ends up less then the low number, it will know you cheated during the game.
+        if (hiNum < lowNum) {
+          console.log("Youre Cheating!")
+          // quit the game in spite.
+          process.exit()
+        }
+
+        // redefine the randNum to account for the new high or low numbers.
+        randNum = randomInteger(lowNum, hiNum)
+        
+        // generate a new number to ask if it's that.
+        console.log(randNum)
+          
+      } 
+      // second part of the if statement for if the user says the number is lower.
+      else if (capitalize(highOrLow).charAt(0) === "L") {
+
+        // redefine the new high number to help narrow down the selection
+        hiNum = randNum - 1  
+
+        // if the high number ends up lower then the low number, it will know you cheated.
+        if (hiNum < lowNum) {
+            console.log("Youre Cheating!")
+
+            // quit the game in spite.
+            process.exit()
+          }
+
+        // redefine randNum for creating a new random number using the updated low and high numbers.
+        randNum = randomInteger(lowNum, hiNum)
+
+        // create a new number to ask if this one is right
+        console.log(randNum)
+      }
+
+      // redefine response to continue loop
+      response = await ask("What about this one? ")
+
+      // if they input something that is not a Yes or No to the initial question if that was your number.
+    } else {
+      console.log("That's not an appropriate response.")
+
+      // redefine response to continue loop
+      response = await ask("Please try again.\nWas I right? ")
+    }
+
+
+  }
+  // if the computer wins, it gloats and quits.
+  console.log("I knew it! I'm smarter then you Human!")
+  process.exit()
+}
+
+// to take this farther, you can optomize the guesses so that instead of them being random. it will take the 
+// (high number + low number)/2 = compGuess so that every guess will be in the middle of the selection removing the most
+// possibilities per guess.
```

Comment:
I would say your next step would be to implement this, which it looks like you already have a good grasp of how to do